.github/CODEOWNERS

```diff
@@ -38,3 +38,4 @@
 /uptime/                @jeremy-lq
 /vespa/                 dd@vespa.ai
 /vns3/                  @DataDog/agent-integrations
+/contrastsecurity				kristiana.mitchell@contrastsecurity.com
```

Comment:
Seems you are using tabs here and the rest of the file is using spaces. Make sure your email is aligned with the others